src/main/java/com/melon/portfoliomanager/dtos/UserDto.java

```diff
@@ -0,0 +1,72 @@
+package com.melon.portfoliomanager.dtos;
+
+
+import jakarta.persistence.Column;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+public class UserDto {
+
+    @NotNull(message = "Invalid user input. The username is null.")
+    @Column(name = "username", unique = true)
+    private String username;
+
+    @Email(message = "Invalid user input. The email should be valid.")
+    @NotNull(message = "Invalid user input. The email is null.")
+    @Column(name = "email")
+    private String email;
+
+    @Column(name = "first_name")
+    private String firstName;
+
+    @Column(name = "last_name")
+    private String lastName;
+
+
+    public UserDto(String username, String email, String firstName, String lastName) {
```

Comment:
Is this constructor used anywhere in the moment ? If not, we can safely delete it and when we need it we will create it in the future.